src/widgets/settingspages/NotificationPage.cpp

```diff
@@ -133,4 +134,31 @@ NotificationPage::NotificationPage()
         }
     }
 }
+QComboBox *NotificationPage::createToastReactionComboBox(
+    std::vector<pajlada::Signals::ScopedConnection> managedConnections)
+{
+    QComboBox *toastReactionOptions = new QComboBox();
+
+    for (int i = 0; i <= static_cast<int>(ToastReactions::DontOpen); i++)
+    {
+        toastReactionOptions->insertItem(
+            i, Toasts::findStringFromReaction(static_cast<ToastReactions>(i)));
+    }
+
+    // update when setting changes
+    pajlada::Settings::Setting<int> setting = getSettings()->openFromToast;
+    setting.connect(
+        [toastReactionOptions](const int &index, auto) {
+            toastReactionOptions->setCurrentIndex(index);
+        },
+        managedConnections);
+
+    QObject::connect(toastReactionOptions, &QComboBox::currentTextChanged,
```

Comment:
Couldn't figure out how to properly connect the currentIndexChanged Signal (with int), to update the setting from that.

Reply:
Now the proper event is connected thanks to @pajlada FeelsGoodManClap